newdle/cli.py

```diff
@@ -0,0 +1,57 @@
+from datetime import datetime, timedelta
+
+import click
+from flask import Blueprint, current_app
+from sqlalchemy import or_
+
+from .core.db import db
+from .models import Newdle
+
+
+cli = Blueprint('newdle-cli', __name__, cli_group=None)
+
+
+@cli.cli.command('cleanup-newdles')
+@click.option(
+    '-n',
+    '--dry-run',
+    is_flag=True,
+    help='If enabled, does not commit changes to the DB.',
+)
+def cleanup_newdles(dry_run):
+    """Removes old newdles from the database."""
+    last_activity_cleanup_days = current_app.config['LAST_ACTIVITY_CLEANUP_INTERVAL']
+    final_date_cleanup_days = current_app.config['FINAL_DATE_CLEANUP_INTERVAL']
+    now = datetime.utcnow()
+    if not last_activity_cleanup_days and not final_date_cleanup_days:
+        current_app.logger.warn(
+            'LAST_ACTIVITY_CLEANUP_INTERVAL and FINAL_DATE_CLEANUP_INTERVAL are not set.'
+        )
+        current_app.logger.warn('Nothing to do.')
```

Comment:
I would only use one warning. Otherwise it's quite ugly if we ever start using sentry - then this would be logged as two separate events.